lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosResult.cxx

```diff
@@ -266,7 +266,7 @@ FunctionalChaosResult::FunctionCollection FunctionalChaosResult::getReducedBasis
 }
 
 /* Composed meta model accessor */
-Function FunctionalChaosResult::getComposedMetaModel() const
+DualLinearCombinationFunction FunctionalChaosResult::getComposedMetaModel() const
```

Comment:
This code allows to get the pretty-print, which is attached to the `DualLinearCombinationFunction` class.

Reply:
That was just for the record, not as a suggestion for code improvement.